pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -335,6 +335,13 @@ func (r *ReconcileOperConfig) Reconcile(ctx context.Context, request reconcile.R
 	allResources = append(allResources, deployments...)
 	r.podReconciler.SetResources(allResources)
 
+	// Remove ovnkube-node object if ovn db route doesn't exist in hypershift management cluster
+	if bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Enabled && bootstrapResult.OVN.OVNKubernetesConfig.HypershiftConfig.Route == "" {
```

Comment:
Do we ever need to worry about the route changing or going away? If that happens, what should / could we do?

Reply:
Maybe mark the operator status to be degraded=true if route goes away for certian period of time?
If route changes are detected, ovnk-node will be reconciled, we do nothing?